gm4_rituals/data/rituals/functions/altar/ritual/clear_weather/run.mcfunction

```diff
@@ -0,0 +1,24 @@
+#runs rain ritual
+#@s - @e[type=minecraft:area_effect_cloud,tag=gm4_altar]
+#called by rituals:altar/ritual/thunder/checks
+
+#audiovisuals
+playsound minecraft:ambient.underwater.exit player @a[distance=..20] ~ ~ ~ 100 2 1
+playsound minecraft:ambient.underwater.exit player @a[distance=..20] ~ ~ ~ 100 1 1
+playsound minecraft:ambient.underwater.exit player @a[distance=..20] ~ ~ ~ 100 0 1
+
+particle minecraft:item_snowball ~ ~4 ~ 1 0 1 0 500 normal @a
+particle minecraft:poof ~ ~4 ~ 1 0 1 0 500 normal @a
+
+summon minecraft:lightning_bolt ~ ~-.5 ~
+
+#functionality
+## clear weather - unspecified time
+weather clear
```

Comment:
Time should be specified. The time will now depend on the seed and the weather cycle will start over. This means that with an unlucky seed, it could start raining in a few minutes.
Setting the weather to `rain 1` is a better solution.